pandas/core/strings.py

```diff
@@ -468,11 +493,20 @@ def f(x):
 
 def str_get_dummies(arr, sep='|'):
     """
-    Split each string by sep and return a frame of dummy/indicator variables.
+    Split each string in the ``Series`` by sep and return a frame of dummy/indicator variables.
```

Comment:
Can you keep to PEP8 line length? (for docstring it is actually even 72 to be strict) -> this is important because eg ipython pop-up wraps long lines giving a strange effect